ReactQt/runtime/src/reactplugin.cpp

```diff
@@ -14,6 +14,13 @@
 #include "reactplugin.h"
 #include "utilities.h"
 
+#include <QLoggingCategory>
+
 void ReactPlugin::registerTypes(const char* uri) {
     utilities::registerReactTypes();
+    QLoggingCategory::setFilterRules("UIManager=true\n"
```

Comment:
Should it be removed before merge?

Reply:
I found it to be quite convenient when you can change logger settings in code instead of setting env var.
Maybe we can leave it commented for debug purpose, wdyt?

Reply:
sounds good, or maybe under #ifdef _DEBUG condition